inter-operator/architecture/basic.md

```diff
@@ -531,6 +545,73 @@ status:
 
 `Unbind` is handled using Kubernetes [finalizers](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#finalizers).
+
+
+# Multi-Cluster provisioning Support for Interoperator
+Multi-cluster provisioning support enables provisioning and distribution of the service instances into multiple clusters. Amongst the multiple list of clusters, one is selected based on the scheduler chosen and the `SFServiceInstance` updated with the `clusterId` value and is also copied in the cluster it is assigned/belongs to. Every cluster should have service operator also installed within it and the service fabrik inter-operator provisioner would then pick up the event generated by the creation of the `SFServiceInstnce` which in turn creates the service specififc CRDs which service operator listens to.
+## Why Multi Cluster Support is needed
+Scalability is the main reason why one should use Multi-Cluster support. It gives you an option to add new clusters into your set of clusters and scale horizontally. There is always a limitation with the number of resources you can spawn in a cluster, even with maximum number of workers attached to a cluster, one can only scale to a finite number of worker node, which can hold a finite number of services(resources) only. Even with multiple worker nodes, APIServer(ETCD) also can become the bollteneck for scaling. Hence, for a production sceranio, multi-cluster support will be required so that services can be scheduled and spread across multiple clusters and can be scaled horizontally. Regarding the type of scheduling algorithms which are supported, we currently support round-robin and least-filled scheduler. We also plan to implement other schedulers which can be used. Schedulers are discussed later in the [schedulers](#schedulers) section.
+## New Custom Resources Introduced
+Along with the new custom resources like `SFService`, `SFPlan`, `SFServiceInstance` and `SFServiceBinding` which are discussed earlier, we also introduce `SFCluster` as a new CRD.
+### SFCluster
+`SFCluster` is the CRD which stores the details of the cluster where provisioning of the service is needed. Depending on how many clusters need to be onboarded, we need to add resources for those. The structure of a sample resource look like the following.
+
+```yaml
+apiVersion: resource.servicefabrik.io/v1alpha1
+kind: SFCluster
+metadata:
+  name: "1"
+  namespace: interoperator
+spec:
+  secretRef: 1-kubeconfig
+```
+where the secretRef looks like the following
+
+```yaml
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: vivek2-kubeconfig
```

Comment:
name should be "1-kubeconfig" as the secretRef has that name

Reply:
done